src/Configuration/ConfigDefinition.php

```diff
@@ -25,62 +27,10 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->isRequired()
                     ->cannotBeEmpty()
                 ->end()
-                ->append($this->addDbNode())
-                ->append($this->addTablesNode())
+                ->append(DbNode::create())
+                ->append(TablesNode::create())
```

Comment:
dtto, should also be injected in constructor (maybe nullable and create this if null)